osu.Game.Rulesets.Catch.Tests/TestSceneAutoJuiceStream.cs

```diff
@@ -27,9 +27,9 @@ protected override IBeatmap CreateBeatmap(RulesetInfo ruleset)
                 }
             };
 
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < 12; i++)
             {
-                float width = (i % 10 + 1) / 20f * CatchPlayfield.WIDTH;
+                float width = (i + 1) / 20f * CatchPlayfield.WIDTH;
```

Comment:
Is there any particular reason for these test changes?

Reply:
To visually inspect hyper dash trails in a replay (the catcher was not hyper dashing in this test scene).
No separate test case is added because the trail generation is still buggy (multiple hyperdash afterimages displayed, which requires a substantial change of code to address).